app/components/UI/OnboardingWizard/Step3/index.js

```diff
@@ -134,15 +128,8 @@ Step3.propTypes = {
   onClose: PropTypes.func,
 };
 
-const mapStateToProps = (state) => ({
-  accounts: selectAccounts(state),
-  currentCurrency: selectCurrentCurrency(state),
-  selectedAddress: selectSelectedAddress(state),
-  identities: selectIdentities(state),
-});
-
 const mapDispatchToProps = (dispatch) => ({
   setOnboardingWizardStep: (step) => dispatch(setOnboardingWizardStep(step)),
 });
 
-export default connect(mapStateToProps, mapDispatchToProps)(Step3);
+export default connect(mapDispatchToProps)(Step3);
```

Comment:
Hmm I think we need to do something like 
```suggestion
export default connect(null, mapDispatchToProps)(Step3);
```

If i'm reading correctly the connect signature: https://react-redux.js.org/api/connect

Reply:
thank you. addressed in the latest commit.